examples/serverless/variables.tf

```
@@ -33,4 +33,10 @@ variable "sg_name" {
   type        = string
   default     = ""
   description = "Name to give to created security group"
-}
\ No newline at end of file
+}
+
+variable "serverless_snapshot_arns_to_restore" {
+  type        = list(string)
+  default     = []
+  description = "Optional list of snapshot ARNs to restore for serverless ElastiCache."
```

This might be a nitpick but could we update this description to something like:

```suggestion
  description = "The list of ARN(s) of the snapshot that the new serverless cache will be created from. Available for Redis only."
```